pkgs/top-level/all-packages.nix

```diff
@@ -11763,7 +11763,7 @@ with pkgs;
 
   darwin = let
     apple-source-releases = callPackage ../os-specific/darwin/apple-source-releases { };
-  in apple-source-releases // rec {
+  in recurseIntoAttrs (apple-source-releases // rec {
```

Comment:
Not sure if we want to build everything in here, there's some stuff in there that depends on xcode and I don't know if all of the licences are correct.